selfdrive/ui/translations/main_pt-BR.ts

```diff
@@ -474,14 +474,14 @@ trabalho definido</translation>
 </context>
 <context>
     <name>ParamControl</name>
-    <message>
-        <source>Ok</source>
-        <translation>OK</translation>
-    </message>
     <message>
         <source>Cancel</source>
         <translation>Cancelar</translation>
     </message>
+    <message>
+        <source>Enable</source>
+        <translation>Ativar</translation>
+    </message>
```

Comment:
@AlexandreSato does this look right?

Reply:
"ativar" or "habilitar" looks very good!